src/index.js

```diff
@@ -1375,6 +1379,8 @@ let errorCheck = () => {
                 && thisJamPenalties.filter(x => x.skater == leadJammer).length != 0
                 && events.filter(x => x.event == 'lost' && x.skater == leadJammer).length == 0
             ){
+              // If a penalty is assessed to a jammer between jams (check
```

Comment:
I honestly don't know the answer to this, but the current system throws errors on this, which may or may not be correct.